courses/1-ramda-basics/3-currying.js

```diff
@@ -166,8 +168,12 @@ describe("Course 1.3 - Currying", () => {
 
     it("Use curred functions to encode messages", () => {
       const encode = R.pipe(
+        R.map(R.indexOf(R.__, LETTERS)),
```

Comment:
For the sake of learning, please comment here with a custom function version of `R.map(R.indexOf(R.__, LETTERS))` that does not use Ramda. I think that will help cement the behavior of what you're writing. :smile:

Reply:
Here's my javascript function. This was great practice!

`function lettersToNum(str) {
  
  const LETTERS = " abcdefghijklmnopqrstuvwxyz";
  let strToNum = [];
  
  for (let i = 0; i < str.length; i++) {
      let idx = LETTERS.indexOf(str[i]);
    
        strToNum.push(idx);
      
      }
      return strToNum.join("");
      }`

Reply:
Not sure why it's putting some of the code outside the snippet but hopefully it's not too distracting.

Reply:
<pre>```js
Code can be formatted like this :smile:
```</pre>